src/server/slowlog.h

```diff
@@ -0,0 +1,82 @@
+// Copyright 2022, DragonflyDB authors.  All rights reserved.
+// See LICENSE for licensing terms.
+//
+
+#pragma once
+
+#include <boost/circular_buffer.hpp>
+#include <string>
+#include <vector>
+
+#include "base/integral_types.h"
+#include "server/common.h"
+namespace dfly {
+
+class SlowLogEntry {
+ public:
+  SlowLogEntry(uint32_t command_id, uint32_t unix_timestamp, uint64_t execution_time_micro,
+               size_t original_length, std::vector<std::pair<std::string, uint32_t>> cmd_args,
+               std::string client_ip, std::string client_name);
+
+  // returns a vector of pairs of argument and extra bytes if the argument was truncated
+  const std::vector<std::pair<std::string, uint32_t>>& CmdArgs() const {
+    return cmd_args_;
+  }
+
+  uint32_t CommandId() const {
```

Comment:
Ok now I understand.. the command id is the incremental id. This name is confusing because we already have command id for each command in dragonfly (see command registry).
I would rename this to entry_id_